asm/code_8075BA4.s

```diff
@@ -27364,8 +27364,8 @@ _080835FA:
 	bx r1
 	thumb_func_end sub_8083568
 
-	thumb_func_start sub_8083600
-sub_8083600:
+	thumb_func_start GetLeader
```

Comment:
```suggestion
	thumb_func_start GetLeaderEntity
```

Returns a `DungeonEntity`. Up to you on whether this better or not.

Reply:
Done.